diff --git a/arch/arm64/boot/dts/freescale/imx8mm-var-dart-customboard.dts b/arch/arm64/boot/dts/freescale/imx8mm-var-dart-customboard.dts
index bedc178..feab02a 100644
--- a/arch/arm64/boot/dts/freescale/imx8mm-var-dart-customboard.dts
+++ b/arch/arm64/boot/dts/freescale/imx8mm-var-dart-customboard.dts
@@ -35,6 +35,13 @@
 		status = "okay";
 	};
 
+    pwm-beeper {
+        compatible = "pwm-beeper";
+        pwms = <&pwm3 0 5000>;
+        volume-levels = <0 8 20 40 500>;
+        default-volume-level =<4>;
+    };
+
 	gpio-keys {
 		compatible = "gpio-keys";
 		pinctrl-names = "default";
@@ -443,6 +450,12 @@
 	pinctrl-0 = <&pinctrl_pwm1>;
 };
 
+&pwm3 {
+	status = "okay";
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_pwm3>;
+};
+
 &snvs_pwrkey {
 	status = "okay";
 };
-- 
2.25.1

