diff --git a/arch/arm64/boot/dts/freescale/imx8mm-var-dart-customboard.dts b/arch/arm64/boot/dts/freescale/imx8mm-var-dart-customboard.dts
index bedc178..6cef0e1 100644
--- a/arch/arm64/boot/dts/freescale/imx8mm-var-dart-customboard.dts
+++ b/arch/arm64/boot/dts/freescale/imx8mm-var-dart-customboard.dts
@@ -104,7 +104,7 @@
 
 &csi1_bridge {
 	fsl,mipi-mode;
-	status = "okay";
+	status = "disabled";
 	port {
 		csi1_ep: endpoint {
 			remote-endpoint = <&csi1_mipi_ep>;
@@ -356,7 +356,7 @@
 	};
 
 	ov5640_mipi1: ov5640_mipi1@3c {
-		status = "okay";
+		status = "disabled";
 		compatible = "ovti,ov5640_mipi";
 		reg = <0x3c>;
 		pinctrl-names = "default";
@@ -408,7 +408,7 @@
 &mipi_csi_1 {
 	#address-cells = <1>;
 	#size-cells = <0>;
-	status = "okay";
+	status = "disabled";
 	port {
 		mipi1_sensor_ep: endpoint@1 {
 			remote-endpoint = <&ov5640_mipi1_ep>;
diff --git a/arch/arm64/boot/dts/freescale/imx8mm-var-dart.dtsi b/arch/arm64/boot/dts/freescale/imx8mm-var-dart.dtsi
index 50522b8..cbf6fb3 100644
--- a/arch/arm64/boot/dts/freescale/imx8mm-var-dart.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx8mm-var-dart.dtsi
@@ -586,7 +586,7 @@
 };
 
 &mipi_dsi {
-	status = "okay";
+	status = "disabled";
 
 	port@1 {

